CCT_cold/config.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": "Columbia Card Task Cold Version",
+        "run": [
+                 "experiment.js",
+                 "style.css"
+               ],
+        "tag": "columbia_card_task_cold",
```

Comment:
Hmm, my CCT-cold.js file says that the experiment name is columbia_card_task_cold...

Should I be changing it somewhere else?


> On Jan 15, 2016, at 12:14 PM, Vanessa Sochat <notifications@github.com> wrote:
> 
> In CCT_cold/config.json:
> 
> > @@ -0,0 +1,25 @@
> > +[
> > +    {
> > +        "name": "Columbia Card Task Cold Version",
> > +        "run": [
> > +                 "experiment.js",
> > +                 "style.css"
> > +               ],
> > +        "tag": "columbia_card_task_cold",
> here the tag is "columbia_card_task_cold" but the experiment has "CCT_cold"
> 
> —
> Reply to this email directly or view it on GitHub.
>